libs/components/packages/src/schematics/migrations/update-10/axe-core/axe-core.schematic.ts

```diff
@@ -0,0 +1,28 @@
+import { Rule } from '@angular-devkit/schematics';
+import { NodePackageInstallTask } from '@angular-devkit/schematics/tasks';
+import {
+  addPackageJsonDependency,
+  getPackageJsonDependency,
+} from '@schematics/angular/utility/dependencies';
+
+/**
+ * Sets axe-core to ~4.8.3. We can't put this in the ng-update.packageGroup
+ * because we only want to update a customer's axe-core version if they've
+ * installed @skyux-sdk/testing.
+ */
+export default function updateAxeCore(): Rule {
+  return (tree, context) => {
+    if (getPackageJsonDependency(tree, '@skyux-sdk/testing') !== null) {
+      const axeCore = getPackageJsonDependency(tree, 'axe-core');
+
+      if (axeCore !== null) {
```

Comment:
If they have testing but are missing the peer dependency, wouldn't that also be a good reason to install `axe-core`?

Reply:
Good call. Updated.